MultiplayerMod/Platform/Steam/Network/SteamPlayerIdentity.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using MultiplayerMod.Multiplayer;
+using Steamworks;
+
+namespace MultiplayerMod.Platform.Steam.Network;
+
+[Serializable]
+public record SteamPlayerIdentity(CSteamID Id) : IPlayerIdentity {
+
+    public bool Equals(IPlayerIdentity other) {
```

Comment:
Is it need? Records have equals by default.

Reply:
Yeah, but the record adds `Equals(SteamPlayerIdentity)` and the interface requires `Equals(IPlayerIdentity)`🤷‍♂️